src/executable-code/executable-fragment.js

```diff
@@ -555,9 +588,48 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
         }
       }
     });
+
+    /**
+     * Show import suggestions on Alt + Enter
+     */
+    this.codemirror.on('keypress', debounce((mirror, event) => {
+      if (event.altKey && event.keyCode === KEY_CODES.ENTER) {
+        if (this.importsSuggestions.length === 0) return;
+        let cur = mirror.getCursor();
+        let token = mirror.getTokenAt(cur);
+        let interval = {
+          start: {line: cur.line, ch: token.start},
+          end: {line: cur.line, ch: token.end}
+        };
+        let results = this.importsSuggestions
+          .filter( it => JSON.stringify(it.interval) === JSON.stringify(interval) )
+          .map ( it => it.imports )
+          .flat()
+        let withImports = this.canAddImport;
+        if (results.length !== 0) {
+          let options = {
+            hint: function () {
+              return {
+                from: mirror.getDoc().getCursor(),
+                to: mirror.getDoc().getCursor(),
+                list: results.map(result => {
+                  if (!withImports) {
+                    result.import = undefined
```

Comment:
I am not sure, It is so clear for me. Can you explain more?..

Reply:
I had a problem with the addition of imports, if there is a hidden code at the top. So I set the flag, depending on the header, and then, if I can't add it, I ignore the passed imports.